<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Search_Report</class>
 <widget class="QWidget" name="Frm_Search_Report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>评估报告智能检索</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>组合检索条件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="AMModel_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="AMModel">
        <property name="text">
         <string>弹药型号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="2">
       <widget class="QLineEdit" name="ReportName_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLineEdit" name="Comment_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="btn_combination">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>信息查询</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QFontComboBox" name="AMType_2">
        <property name="currentText">
         <string> </string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QCheckBox" name="TargetName">
        <property name="text">
         <string>目标名称</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QCheckBox" name="Comment">
        <property name="text">
         <string>评估结论</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="btn_condition_reset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>条件复位</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="5" colspan="2">
       <widget class="QLineEdit" name="DamageScene_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="ReportNumber_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="DamageDegree">
        <property name="text">
         <string>毁伤等级</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="DamageScene">
        <property name="text">
         <string>毁伤场景</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QFontComboBox" name="DamageDegree_2">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLineEdit" name="TargetName_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="Targets">
        <property name="text">
         <string>打击目标类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ReportCode">
        <property name="text">
         <string>报告编号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QCheckBox" name="AMType">
        <property name="text">
         <string>弹药类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="ReportName">
        <property name="text">
         <string>报告名称</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QFontComboBox" name="Targets_2">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>智能检索条件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>智能化检索支持基于评估报告模糊关键词、部分描述或者作战需求，智能化的匹配符合条件的目标数据条目。</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_nlp_rebuild">
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string>重建索引</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_nlp_search">
        <property name="text">
         <string>智能检索</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLineEdit" name="txt_nlp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>评估报告检索结果</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_noti">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_export">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>导出查询结果</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QTableView" name="tv_result">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
